frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.styled.tsx

```diff
@@ -24,8 +24,8 @@ export const FieldSettingsButtonsContainer = styled.div`
 
 export const WarningBanner = styled.div`
   padding: ${space(2)};
-  border: 1px solid ${color("warning")};
+  border: 1px solid ${() => color("warning")};
```

Comment:
do we need to add a function here?

Reply:
That's explained here. https://github.com/metabase/metabase/pull/43512#discussion_r1627510527

Reply:
awesome, thanks :smile: